Jenkinsfile

```diff
@@ -133,17 +133,41 @@ pipeline {
             when {
                 tag 'v*'
             }
-            steps {
-                JiraIssueNotification(
-                    'DEEP',
-                    'DPM',
-                    '10204',
-                    "[preview-testbed] New Orchestrator version ${env.BRANCH_NAME} available",
-                    "Check new artifacts at:\n\t- Docker image: [${dockerhub_image_id}|https://hub.docker.com/r/${dockerhub_repo}/tags/]",
-                    ['wp3', 'preview-testbed', "orchestrator-${env.BRANCH_NAME}"],
-                    'Task',
-                    'mariojmdavid'
-                )
+            parallel {
+                stage('Notify DEEP') {
+                        steps {
+                                JiraIssueNotification(
+                                    'DEEP',
+                                    'DPM',
+                                    '10204',
+                                    "[preview-testbed] New orchent version ${env.BRANCH_NAME} available",
```

Comment:
Shouldn't it be orchestrator and not orchent?

Reply:
fixed. 
Sorry!